README.md

```diff
@@ -8,435 +8,429 @@ ACL (Access Control List)
-      permissions => [
-       { identity => 'SYSTEM', rights => ['modify'], child_types => 'none' },
-       { identity => 'SYSTEM', rights => ['modify'], child_types => 'containers' },
-       { identity => 'SYSTEM', rights => ['modify'], child_types => 'objects' },
-       { identity => 'SYSTEM', rights => ['full'], affects => 'self_only' },
-       { identity => 'SYSTEM', rights => ['read','execute'], affects => 'direct_children_only' },
-       { identity => 'SYSTEM', rights => ['read','execute'], child_types=>'containers', affects => 'direct_children_only' },
-       { identity => 'SYSTEM', rights => ['read','execute'], child_types=>'objects', affects => 'direct_children_only' },
-       { identity => 'SYSTEM', rights => ['full'], affects => 'children_only' },
-       { identity => 'SYSTEM', rights => ['full'], child_types=>'containers', affects => 'children_only' },
-       { identity => 'SYSTEM', rights => ['full'], child_types=>'objects', affects => 'children_only' },
-       { identity => 'SYSTEM', rights => ['read'], affects => 'self_and_direct_children_only' },
-       { identity => 'SYSTEM', rights => ['read'], child_types=>'containers', affects => 'self_and_direct_children_only' },
-       { identity => 'SYSTEM', rights => ['read'], child_types=>'objects', affects => 'self_and_direct_children_only' }
-      ],
-      inherit_parent_permissions => 'false',
-    }
-```
+With Windows, you can specify the same `identity` with different inheritance and propagation. Each of the resulting items is managed as a separate ACE.
+
+~~~
+acl { 'c:/tempperms':
+  purge                      => 'true',
+  permissions                => [
+   { identity => 'SYSTEM', rights => ['modify'], child_types => 'none' },
+   { identity => 'SYSTEM', rights => ['modify'], child_types => 'containers' },
+   { identity => 'SYSTEM', rights => ['modify'], child_types => 'objects' },
+   { identity => 'SYSTEM', rights => ['full'], affects => 'self_only' },
+   { identity => 'SYSTEM', rights => ['read','execute'], affects => 'direct_children_only' },
+   { identity => 'SYSTEM', rights => ['read','execute'], child_types=>'containers', affects => 'direct_children_only' },
+   { identity => 'SYSTEM', rights => ['read','execute'], child_types=>'objects', affects => 'direct_children_only' },
+   { identity => 'SYSTEM', rights => ['full'], affects => 'children_only' },
+   { identity => 'SYSTEM', rights => ['full'], child_types=>'containers', affects => 'children_only' },
+   { identity => 'SYSTEM', rights => ['full'], child_types=>'objects', affects => 'children_only' },
+   { identity => 'SYSTEM', rights => ['read'], affects => 'self_and_direct_children_only' },
+   { identity => 'SYSTEM', rights => ['read'], child_types=>'containers', affects => 'self_and_direct_children_only' },
+   { identity => 'SYSTEM', rights => ['read'], child_types=>'objects', affects => 'self_and_direct_children_only' }
+  ],
+  inherit_parent_permissions => 'false',
+}
+~~~
+
+
+##Reference
```

Comment:
I see what you're saying @ferventcoder , but Reference at the end is a pretty standard convention for tech doc. Generally, reference information is the kind of information people don't read, per se; they tend to search for specific things instead. As such, reference information works well near the end of the architecture: it's there, it's findable and searchable, but it's not in the way of the meatier "how-to" information.

Beyond that, I think it works well in our module readmes, basically for the same reasons. Our users are used to finding it there, for one thing. Also, our users get a lot of information from the Usage section, and I'm disinclined to make them trudge through a list of parameters when they're actually just trying to remember how to set something up. Some of our parameter lists get quite long!

You're right that user who aren't learning will spend more time in Reference, but we don't want to overwhelm new users, either. Established users are much more likely to just search for the parameter they're looking for, so they are going to pop right down to what they want pretty quickly. And for newer users, the Usage examples can give needed context to the type/class/parameter lists that come after.